Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivityViewModel.kt

```diff
@@ -56,14 +57,16 @@ class MainActivityViewModel @AssistedInject constructor(
     personCertificatesProvider: PersonCertificatesProvider,
     valueSetRepository: ValueSetsRepository,
     tracingSettings: TracingSettings,
-    installTimeProvider: InstallTimeProvider,
 ) : CWAViewModel(
     dispatcherProvider = dispatcherProvider
 ) {
 
-    val isToolTipVisible: LiveData<Boolean> = onboardingSettings.fabScannerOnboardingDone
-        .map { done -> done.not() && installTimeProvider.isInstallFromUpdate }
-        .asLiveData2()
+    val isToolTipVisible: LiveData<Boolean> = combine(
+        onboardingSettings.fabScannerOnboardingDone,
+        onboardingSettings.fabUqsLogVersion
+    ) { done, version ->
+        !done && version != DEFAULT_APP_VERSION && version < BuildConfigWrap.VERSION_CODE
```

Comment:
Shouldn't be this version == DEFAULT_APP_VERSION? Because on fresh install the tooltip isn't currently shown

Reply:
it is intended to `not` show it on fresh install to avoid overwhelming the user with many pop-ups , the idea is to show it on App updates only